quic/s2n-quic-transport/src/recovery/manager/tests.rs

```diff
@@ -1902,7 +1902,7 @@ fn persistent_congestion() {
     // part of the persistent congestion period.
     context.path_mut().rtt_estimator.update_rtt(
         Duration::from_millis(10),
-        Duration::from_millis(700),
+        Duration::from_millis(600),
```

Comment:
Why did this need to change?

Reply:
As the logic of the RTT calculation has been changed, there are some minor updates on testing results for checking the logic.  Some variables such as persistent_congestion_threshold become slightly bigger. I think this change can suppress the errors in tests due to this small inflation.